build_backend/test_onbuild.py

```diff
@@ -38,7 +39,13 @@ def build(request: pytest.FixtureRequest, tmp_path: Path, template_fields: dict)
     (pkg_dir / "streamlink").mkdir(parents=True)
     shutil.copy(PROJECT_ROOT / "pyproject.toml", tmp_path / "pyproject.toml")
     shutil.copy(PROJECT_ROOT / "setup.py", tmp_path / "setup.py")
-    shutil.copy(PROJECT_ROOT / "src" / "streamlink" / "_version.py", pkg_dir / "streamlink" / "_version.py")
+
+    # Lookup for the path from the root source directory
+    if os.path.isfile(PROJECT_ROOT / "src" / "streamlink" / "_version.py"):
+        shutil.copy(PROJECT_ROOT / "src" / "streamlink" / "_version.py", pkg_dir / "streamlink" / "_version.py")
+    else:
```

Comment:
Please disable the branch coverage on the else-block:
`  # pragma: no cover`
https://coverage.readthedocs.io/en/latest/excluding.html#branch-coverage

The codecov config is currently not set up correctly (oops), which is why the PR passed...
https://github.com/streamlink/streamlink/blob/6.7.2/.codecov.yml#L47-L51
https://github.com/streamlink/streamlink/pull/5933/checks?check_run_id=23636174400